src/promptflow/tests/sdk_cli_test/e2etests/test_cli.py

```diff
@@ -1207,7 +1205,6 @@ def test_flow_build_with_ua(self):
                 "--user-agent",
                 "test/1.0.0",
             )
-        assert "not exist" in str(e.value)
```

Comment:
Why removed this check?

Reply:
What's current error message?

Reply:
Because it is exit (1), the error type is SystemExit and there is no message, this is what we expect, when the cli command encounters a Promptflow error, it will not output any traceback information.
![image](https://github.com/microsoft/promptflow/assets/23182548/3aa1b46b-7b00-4973-9859-83fa11ff1d1d)